<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrzadzeniaLista</class>
 <widget class="QMainWindow" name="UrzadzeniaLista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista Urzadzeń</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Skasuj</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDaty">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>1900:01:01</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>188</width>
       <height>68</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelDzien">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>Poniedziałek</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>190</width>
        <height>35</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>188</width>
         <height>33</height>
        </rect>
       </property>
       <widget class="QLabel" name="labelZegara">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>8</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>12:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <zorder>scrollArea_2</zorder>
     <zorder>labelDzien</zorder>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>551</width>
      <height>371</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Odświeź</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Skasuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edytuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Przypisz Kontrahenta</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>scrollArea</zorder>
   <zorder>tableView</zorder>
   <zorder>labelTest</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>labelDaty</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edycja</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInformacje">
    <property name="title">
     <string>Informacje</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionOpcje"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
   <addaction name="menuInformacje"/>
   <addaction name="menuUstawienia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpcje">
   <property name="text">
    <string>Opcje</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
